scripts/zlib_shared/1.2.8/.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: cpp
+
+sudo: false
+
+compiler: clang
+
+matrix:
+  include:
+    - os: linux
+      env: MASON_PLATFORM_VERSION=cortex_a9
+
```

Comment:
Let's also build on linux x86_64 so that we have another platform that tests this package.